### DISCUSSION TITLE
Support 'States' when the United States is selected under 'Tax Rules'

### DISCUSSION THREAD

**Discussion by yvovandoorn** [pretix/pretix#3051] [Ideas]
Currently the Tax Rules section only considers taxation at a country level, however the US (political remarks/thoughts aside) has a very different system. They allow each State to charge tax rates, which can change per locale within a state, but I'd consider that scope creeped further than the idea. 

This idea would allow someone to specify a tax rule for a specific state that would be matched against the state of the registrant. So if CA has a tax rule, then anyone purchasing a ticket with a CA address is subject to the tax. Currently thats not feasible as the logic would mean anyone from IL purchasing a ticket for an event that is having its financials handled in CA would be charged the CA tax rate.

**Comment by raphaelm** [comment]
Yeah, that makes sense, I think we have considered it before at some point.

Is it really dependent on the state of the registrant, though? I vaguely remember that it depends on the state where the event, i.e. the service that is sold, takes place.

**Reply by yvovandoorn** [reply]
I Am Not An US tax professional, who truly feel like they're the experts of making things complicated, but in a specific event I'm working on at the moment -- devopsdays Chicago -- the event is taking place in Illinois but the billing / monies is handled by an organization out of Texas called Conference Ops (https://www.conferenceops.com/). The owner of Conference Ops said they have to only charge sales tax for those registering out of Texas. So right now we're using a custom question in Pretix to determine if their primary address is in Texas and then we utilize an export to CSV/XLS report in order to determine who has responded 'yes' to it and eat the cost. 

Now I'm sure this can differ per state / per event, hence why a V1 of this lets the organizer determine best course of action. No different than what Pretix does today with regards to reverse taxation, etc in Europe (where it displays the very valid warnings that you should consult a tax professional).

**Reply by raphaelm** [reply]
Yup, makes sense!
